src/Hl7.Fhir.Support.Poco/Specification/Terminology/ITerminologyService.cs

```diff
@@ -22,29 +22,37 @@ public interface ITerminologyService
         /// <param name="useGet"> Use the GET instead of POST Http method</param>
         /// <returns>Output parameters containing the result of the operation</returns>
         /// <exception cref="FhirOperationException">Thrown when the terminology service encounters an error</exception>
-        /// <remarks>See http://hl7.org/valueset-operations.html#validate-code for more information</remarks>
+        /// <remarks>See  for more information</remarks>
+        /// <remarks>This function corresponds to the <seealso href="http://hl7.org/fhir/valueset-operation-validate-code.html">$validate-codes</seealso> operation.</remarks>
         Task<Parameters> ValueSetValidateCode(Parameters parameters, string id = null, bool useGet = false);
 
         /// <summary>
-        /// Validate that a coded value is in the code system.
+        /// Test the subsumption relationship between code/Coding A and code/Coding B given the semantics of subsumption in the underlying code system
         /// </summary>
         /// <param name="parameters">Input parameters for the operation</param>
-        /// <param name="id">Id of a specific CodeSystem which is used to validate against</param>
+        /// <param name="id">Id of the code system in which subsumption testing is to be performed.</param>
         /// <param name="useGet">Use the GET instead of POST Http method</param>
-        /// <returns>Output parameters containing the result of the operation</returns>
+        /// <returns>Output parameters containing the subsumption relationship between code/Coding "A" and code/Coding "B".</returns>
         /// <exception cref="FhirOperationException">Thrown when the terminology service encounters an error</exception>
-        Task<Parameters> CodeSystemValidateCode(Parameters parameters, string id = null, bool useGet = false);
+        /// <remarks>This function corresponds to the <seealso href="http://hl7.org/fhir/codesystem-operation-subsumes.html">$subsumes</seealso> operation.</remarks>
+        Task<Parameters> Subsumes(Parameters parameters, string id = null, bool useGet = false);
+    }
 
+    /// <summary>
+    /// Subset of the FHIR terminology service (http://hl7.org/fhir/terminology-service.html) that concerns lookup and validation of single codes.
+    /// </summary>
+    public interface ICodeSystemTerminologyService : ICodeValidationTerminologyService
```

Comment:
```suggestion
    public interface ICodeSystemTerminologyService
```

Reply:
`ICodeSystemTerminologyService` should not be dependent on `ICodeValidationTerminologyService`